tailwind.config.js

```diff
@@ -15,6 +27,13 @@ export default {
         interBold: ["Inter-bold", "sans-serif"],
         interExtraBold: ["Inter-extraBold", "sans-serif"],
         interBlack: ["Inter-black", "sans-serif"],
+        ibmMedium: ["IBM-medium", "monospace"],
+        ibmSemi: ["IBM-semi", "monospace"],
```

Comment:
We should define only the inter font-family and use tailwind's font-weight class (font-bold, font-semibold, font-600, font-900, etc) where we need a variation of this font-family